tests/namespace/test_errors_namespace.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from pyvista.core.errors import PyVistaDeprecationWarning
+
+
+@pytest.mark.parametrize(
+    'name',
+    [
+        'AmbiguousDataError',
+        'DeprecationError',
+        'MissingDataError',
+        'NotAllTrianglesError',
+        'PointSetCellOperationError',
+        'PointSetDimensionReductionError',
+        'PointSetNotSupported',
+        'PyVistaDeprecationWarning',
+        'PyVistaEfficiencyWarning',
+        'PyVistaFutureWarning',
+        'PyVistaPipelineError',
+        'VTKVersionError',
+    ],
+)
+def test_core_errors_namespace(name):
+    with pytest.warns(
+        PyVistaDeprecationWarning, match='now imported as: `from pyvista.core.errors import'
```

Comment:
Purity nitpick:
```suggestion
        PyVistaDeprecationWarning, match=r'now imported as: `from pyvista\.core\.errors import'
```
And should we not match with the name while we're at it?
```suggestion
        PyVistaDeprecationWarning, match=rf'now imported as: `from pyvista\.core\.errors import {name}`\.'
```